src/com/lftechnology/java/training/srijan/basics/coreapp/dao/EmployeeDaoImpl.java

```diff
@@ -0,0 +1,390 @@
+				if (con != null) {
+					con.close();
+				}
+			}
+			catch (SQLException e1) {
+				LOGGER.log(Level.INFO, "SQLException{0}", e1);
+			}
+
+		}
+		return UserRole.INVALID;
+	}
+
+	/**
+	 * <p>Displays details of all employees</p>
+	 * 
+	 * @author srijan
+	 */
+	@Override
+	public List<Employee> viewEmployeeDetails() {
+
+		PreparedStatement statement = null;
+		ResultSet resultSet = null;
+		Connection con = null;
+		List<Employee> employees = new ArrayList<>();
+		con = DbConnection.dbConnection();
+		if (con != null) {
+			String query =
+				"Select * from employeeDetails where is_terminated = ?";
+			try {
+				statement = con.prepareStatement(query);
+				statement.setBoolean(1, false);
+				resultSet = statement.executeQuery();
+				while (resultSet.next()) {
+					Employee employee = new Employee();
+					employee.setId(resultSet.getInt("id"));
+					employee.setUserName(resultSet.getString("username"));
+					employee.setFullName(resultSet.getString("fullName"));
+					employee.setDepartment(resultSet.getString("department"));
+					employee.setAddress(resultSet.getString("address"));
+					employee.setIsTerminated(
+						resultSet.getBoolean("is_terminated"));
+					employees.add(employee);
+				}
+			}
+			catch (SQLException e1) {
+				LOGGER.log(Level.INFO, "SQL Exception{0}", e1);
+			}
+			finally {
+				try {
+					statement.close();
+					con.close();
+				}
+				catch (SQLException e1) {
+					LOGGER.log(Level.INFO, "SQL Exception{0}", e1);
+				}
+			}
+
+		}
+		return employees;
+	}
+
+	@Override
+	public void terminate(int idToTerminate) {
+
+		Connection con = null;
+		ResultSet resultSet = null;
+		con = DbConnection.dbConnection();
+		PreparedStatement prepareStatementUpdateInfo = null;
+		PreparedStatement prepareUpdate = null;
+		try {
+			if (con != null) {
+				String query = "Select * from employeeDetails where id = ?";
+				prepareStatementUpdateInfo = con.prepareStatement(query);
+				prepareStatementUpdateInfo.setInt(1, idToTerminate);
+				resultSet = prepareStatementUpdateInfo.executeQuery();
+				while (resultSet.next()) {
+					Boolean status = resultSet.getBoolean("is_terminated");
+					status = !status;
+					String queryUpdate =
+						"UPDATE employeeDetails SET is_terminated = ? where id = ? ";
+					prepareUpdate = con.prepareStatement(queryUpdate);
+					prepareUpdate.setBoolean(1, status);
+					prepareUpdate.setInt(2, idToTerminate);
+					prepareUpdate.executeUpdate();
+				}
+			}
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception:{0}", ex);
+		}
+		finally {
+			try {
+				resultSet.close();
+				prepareStatementUpdateInfo.close();
+				prepareUpdate.close();
+				con.close();
+			}
+			catch (SQLException ex) {
+				LOGGER.log(Level.INFO, "SQL Exception {0}", ex);
+			}
+		}
+	}
+
+	/**
+	 * <p>Find user by username</p>
+	 * 
+	 * @param userName
+	 * @return
+	 * @author srijan
+	 */
+	public Employee findUser(String userName) {
+
+		System.out.println("profile");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)